bundles/CoreBundle/src/Migrations/Version20230222174636.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pimcore\Bundle\CoreBundle\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20230222174636 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        if($schema->getTable('documents_page')->hasColumn('metaData')) {
+            $this->addSql('ALTER TABLE documents_page DROP COLUMN metaData');
+        }
+
+    }
+
+    public function down(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE documents_page ADD COLUMN `metaData` TEXT');
```

Comment:
please add a check that is not already existing before altering

```suggestion
        $this->addSql('ALTER TABLE documents_page ADD COLUMN `metaData` TEXT AFTER `description`');
```